packages/jaeger-ui/src/utils/prefix-url.test.js

```diff
@@ -37,3 +37,25 @@ describe('prefixUrl()', () => {
     });
   });
 });
+
+describe('getPathPrefix()', () => {
+  const tests = [
+    { address: '2a00:8a00:4000:751e::1c:8443' },
+    { address: '[2a00:8a00:4000:751e::1c]:8443' },
+    { address: '[2a00:8a00:4000:751e:0000:0000:0000:001c]:8443' },
+    { address: '[::]:443' },
+    { address: '127.0.0.1' },
+    { address: '127.0.0.1:8443' },
+    { address: 'jaeger.mydomain.com:8443' },
+    { address: 'jaeger.mydomain.com' },
+  ];
+
+  tests.forEach(({ address }) => {
+    it(`getPathPrefix "${address}" correctly`, () => {
```

Comment:
the test name should read as a sentence, so replace `getPathPrefix` with `handles` and it will read:
```
it(`handles "${address}" correctly`, () => {
```